tools/code-owners-parser/Azure.Sdk.Tools.RetrieveCodeOwners/Program.cs

```diff
@@ -5,36 +5,47 @@
 namespace Azure.Sdk.Tools.RetrieveCodeOwners
 {
     /// <summary>
-    /// The tool command to retrieve code owners.
+    /// See Program.Main comment.
     /// </summary>
     public static class Program
     {
         /// <summary>
-        /// Retrieves CODEOWNERS information for specific section of the repo
+        /// Given targetPath and CODEOWNERS file path or https url codeownersFilePathOrUrl,
+        /// prints out to stdout owners of the targetPath as determined by the CODEOWNERS data.
         /// </summary>
-        /// <param name="codeOwnerFilePath">The path of CODEOWNERS file in repo</param>
-        /// <param name="targetDirectory">The directory whose information is to be retrieved</param>
-        /// <param name="filterOutNonUserAliases">The option to filter out code owner team alias.</param>
-        /// <returns>Exit code</returns>
-
+        /// <param name="targetPath">The path whose owners are to be determined.</param>
+        /// <param name="codeownersFilePathOrUrl">The https url or path to the CODEOWNERS file.</param>
+        /// <param name="excludeNonUserAliases">Whether owners that aren't users should be excluded from the
+        /// returned owners.</param>
+        /// <returns>
+        /// On STDOUT: The JSON representation of the matched CodeownersEntry.
+        /// "new CodeownersEntry()" if no path in the CODEOWNERS data matches.
+        /// <br/><br/>
+        /// From the Main method: exit code. 0 if successful, 1 if error.
+        /// </returns>
         public static int Main(
-            string codeOwnerFilePath,
-            string targetDirectory,
-            bool filterOutNonUserAliases = false
-            )
+            string targetPath,
```

Comment:
Just keep in mind that you will need to update places like https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/get-codeowners.ps1 when you update to the latest package version.

Reply:
@weshaggard yeah I already have a PR for that (in draft state as of now), thanks for the reminder! :)
- #5103